simpeg/maps/_property_maps.py

```diff
@@ -963,7 +963,8 @@ def adj(v):
 
         if v is not None:
             return LinearOperator(shp, matvec=fwd, rmatvec=adj) * v
-        return LinearOperator(shp, matvec=fwd, rmatvec=adj)
+        # return LinearOperator(shp, matvec=fwd, rmatvec=adj)
+        return sp.hstack([sdiag(np.ones(nC)), 1j * sdiag(np.ones(nC))])
```

Comment:
This is not technically the same as the previous one, the transpose is not the same :/.